### DISCUSSION TITLE
Add Consistent Help-Switching to ZOAU Commands

### DISCUSSION THREAD

**Discussion by EnsonoStevenPerva** [IBM/zoau-samples#19] [Ideas] [answered]
Not sure if this belongs here or elsewhere, but I would like to propose adding consistent help switches to the ZOAU commands. The documentation calls out both `-h` and `-?` as viable help switches but I have found, in teaching others about ZOAU, that the results are not consistent from command to command.

The commands that do not respond appropriately to a `-?` switch:

- ddls
- decho
- dwhence
- jcan
- jls
- jsub
- mvscmd
- mvscmdauth
- mvstmp
- opercmd
- pcon
- pjdd

I am using ZOAU version 1.2.0, the August 5th build, and have checked every documented command for consistency.

**Comment by MikeFultonDev** [comment, marked as answer]
Thanks @EnsonoStevenPerva . It's a good point. Seems reasonable to tackle in a future release - consistency is important for this type of thing.

**Reply by AnthonyGiorgio** [reply]
This will be addressed in the upcoming ZOAU v1.2.1 release.

**Comment by ddimatos** [comment]
This is has been recorded in JIRA NAZARE-8852, thank you.
